packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -589,7 +589,11 @@ export default class QuanticGeneratedAnswer extends LightningElement {
   }
 
   get shouldDisplayCustomNoAnswerMessage() {
-    return this.state?.cannotAnswer && this.hasCustomNoAnswerMessage;
+    return (
+      this.state?.cannotAnswer &&
+      this.searchStatusState?.hasResults &&
+      this.hasCustomNoAnswerMessage
+    );
```

Comment:
Did we have the same issue in Atomic @lvu285 ?

Reply:
yes they did the ticket is here: https://coveord.atlassian.net/browse/SVCC-4848